record/record.controller.js

```diff
@@ -94,7 +96,21 @@
         };
 
         vm.sharePopup = function() {
+            var permalink;
             var tuple = $rootScope.tuple;
+
+            if (chaiseConfig.resolverImplicitCatalog !== undefined && chaiseConfig.resolverImplicitCatalog !== null) {
+                var catalogFilter = "";
+                // catalogId in chaiseConfig property does not match current catalogId
+                if (chaiseConfig.resolverImplicitCatalog != $rootScope.reference.location.catalog) catalogFilter = chaiseConfig.resolverImplicitCatalog + "/";
+                // catalogId in chaiseConfig property is 'false'
+                if (chaiseConfig.resolverImplicitCatalog == false) catalogFilter = $rootScope.reference.location.catalog + "/";
+                // TODO should this be RID or uniqueId?
+                permalink = $window.location.origin + "/id/" + catalogFilter + tuple.data.RID;
```

Comment:
I know RID will always be available with the current version of ermrest, but we should actually check for it first and if it doesn't exist, we should not offer resolver.